oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -42,13 +48,15 @@ public class FlatFileNodeStoreBuilder {
 
     private static final String FLAT_FILE_STORE_DIR_NAME_PREFIX = "flat-fs-";
 
-    public static final String OAK_INDEXER_USE_ZIP = "oak.indexer.useZip";
+    public static final String OAK_INDEXER_USE_ZIP = "oak.indexer.compressionEnabled";
```

Comment:
@Ewocker might we should not change the property key here for risk of compatibility if which is used somewhere already.

Reply:
done